assets/src/edit-story/app/media/utils/index.js

```diff
@@ -14,13 +14,9 @@
  * limitations under the License.
  */
 
-export { default as createResource } from './createResource';
-export { default as getFirstFrameOfVideo } from './getFirstFrameOfVideo';
 export { default as getResourceFromAttachment } from './getResourceFromAttachment';
 export { default as getResourceFromLocalFile } from './getResourceFromLocalFile';
 export { default as getResourceFromMediaPicker } from './getResourceFromMediaPicker';
 export { default as getResourceFromMedia3p } from './getResourceFromMedia3p';
```

Comment:
^ Should these also be moved?

Reply:
Good question. In @swissspidy POC didn't bother. We can always move them later, but these are not used outside of editor.